.gitattributes

```diff
@@ -1,9 +1,10 @@
 hack/verify-flags/known-flags.txt merge=union
 test/test_owners.csv merge=union
 
-**/zz_generated.*.go -diff
-**/types.generated.go -diff
-**/generated.pb.go -diff
-**/generated.proto -diff
-**/types_swagger_doc_generated.go -diff
-docs/api-reference/** -diff
+**/zz_generated.*.go -diff linguist-generated=true
+**/types.generated.go -diff linguist-generated=true
+**/generated.pb.go -diff linguist-generated=true
+**/generated.proto -diff linguist-generated=true
+**/types_swagger_doc_generated.go -diff linguist-generated=true
+docs/api-reference/** -diff linguist-generated=true
+api/swagger-spec/*.json -diff linguist-generated=true
```

Comment:
I'm not 100% sure but I think we want to see this in review.

Reply:
Are you...sure? This opens up several hundred lines of json on the top of https://github.com/kubernetes/kubernetes/pull/53988/files at least. Are most reviewers actually reading those json dumps? I was under the impression most folks look at the `.go` files where these are defined in more readable form. 

In either case - these aren't lost from the PR - they just default to collapsed.

Reply:
ah OK I am ok with hiding that by default.